scripts/import-pr.sh

```diff
@@ -0,0 +1,55 @@
+#!/bin/bash
+
+if [ -z "$1" ]; then
+  echo "Please provide the full branch URL as an argument."
+  exit 1
+fi
+
+if [ -z "$2" ]; then
+    echo "Please provide the full pull request URL as second argument."
+    exit 1
+fi
+
+# Extract PR number from the URL
+PR_URL="$2"
+PR_NUMBER=$(echo "$PR_URL" | awk -F '/' '{print $NF}')
+
+# Fetch PR Info
+PR_TITLE=$(gh pr view "$PR_NUMBER" --json title -q '.title')
+PR_DESCRIPTION=$(gh pr view "$PR_NUMBER" --json body -q '.body')
+BRANCH_NAME=$(gh pr view "$PR_NUMBER" --json headRefName -q '.headRefName')
+
+# Extracting the repo URL and branch name
+FULL_URL="$1"
+REPO_URL=$(echo "$FULL_URL" | awk -F "/tree/" '{print $1}')
+REPO_NAME=$(basename "$REPO_URL")
+NEW_REMOTE_URL="https://github.com/status-im/status-mobile"
+
+# Clone the repo if not already cloned
+if [ ! -d "$REPO_NAME" ]; then
+    git clone "$REPO_URL"
```

Comment:
Why are we cloning the repo? I don't get it? You know you can just checkout a PR branch directly from `origin`?
```
git fetch origin pull/$ID/head:$BRANCHNAME
git checkout $BRANCHNAME
```
https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/checking-out-pull-requests-locally